bfx-pay-woocommerce.php

```diff
@@ -1,17 +1,18 @@
 <?php
 /**
  * @wordpress-plugin
- * Plugin Name:       Bitfinex Pay
- * Plugin URI:        https://github.com/bitfinexcom/bfx-pay-woocommerce/
+ * Plugin Name:       Bitfinex Pay Woocommerce Plugin
```

Comment:
can you please remove this naming, they've rejected such name for us
it should be `Bitfinex Pay`